### DISCUSSION TITLE
Welcome to caltopo_python Discussions!

### DISCUSSION THREAD

**Discussion by caver456** [ncssar/caltopo_python#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by daveb-arc** [comment]
David here -- just joined because I'm talking w/ WY SAR that is using CalTopo and Eagle Eyes for Drone control.  Working with the Caltopo API and reviewing this Python project.

Will be at the WY SAR Conference next month to chat with SAR agencies about CalTopo, Eagle Eyes and this CalTopo project.
https://www.uwyo.edu/worth/events/sars.html

Anything you want me to share at the SAR conference let me know.

**Reply by caver456** [reply]
Thanks for reaching out.  Nothing specific comes to mind, but it will be interesting to hear if/how folks there are thinking of making use of this project.  We use radiolog (another project in this repo) heavily, which makes use of caltopo_python to create or move gps-enabled radio locators on the map as actual marker objects.  We use radiolog on 100% of callouts since it went live in 2015.  We do have a few other projects that rely on caltopo_python too - but I'm not sure how much it's used outside of our team.  Also I believe our  drone team makes use of it, but I'm not too familiar with those details - let me know if you'd like to get in touch with our drone team tech folks.

**Reply by daveb-arc** [reply]
Would be great to chat w/ the drone team --I would like to align as much as possible with existing SAR processes that are being used in the field.  I'll work on getting the caltopo_python and radiolog (https://github.com/ncssar/radiolog) in my test environment to show to WY SAR teams.

**Reply by caver456** [reply]
Talked with the drone techie - it's an app he's working on, but, it doesn't use caltopo_python and it's still in development.  So I was misleading you with the drone connection.  Still - glad to hear you're looking at radiolog.  Plans console is another repo here that uses caltopo_python.  We have some other repos that haven't seen much use in a while that do use caltopo_python v1.

**Reply by dboyceman** [reply]
I'm working on an app too so would be good to collaborate.  I am building the solution in Microsoft cloud with PowerApp as the secure UI portal.  Added support for Garmin devices via Garmin APIs and two way messaging -- all working in conjunction with CalTopo.  Business logic, alerting and messaging in secure PowerApp and mapping with drone control via caltopo / Eagle Eyes via Teams API.

**Comment by caver456** [comment]
Let me find the right way to get our drone techie in touch with you.

**Comment by daveb-arc** [comment]
Thanks

David Boyce
Principal Consultant

M: 206.679.0114<tel:2066790114>
W: www.adventurereadyconsulting.com<http://www.adventurereadyconsulting.com/>
E: ***@***.******@***.***>
L: www.linkedin.com/in/dboyceman<http://www.linkedin.com/in/dboyceman>

On Apr 17, 2026, at 9:38 AM, Tom Grundy ***@***.***> wrote:

﻿

Talked with the drone techie - it's an app he's working on, but, it doesn't use caltopo_python and it's still in development. So I was misleading you with the drone connection. Still - glad to hear you're looking at radiolog. Plans console is another repo here that uses caltopo_python. We have some other repos that haven't seen much use in a while that do use caltopo_python v1.

**Comment by daveb-arc** [comment]
HI Tom,

Would be interested in working w/ your drone techie on the app — if you can pass my contact info to him if he is interested.

I just saw a City of Durango, CO RFP for real-time radio transcription.  Is this a project you guys would want to work w/ me on?  Good way to grow the radiolog project.

David.

[horizontal bar]

David Boyce
Principal Consultant
ADVENTURE READY CONSULTING

M: 206.679.0114<tel:2066790114>
W: www.adventurereadyconsulting.com<http://www.adventurereadyconsulting.com/>
E: ***@***.******@***.***>
L: www.linkedin.com/in/dboyceman<http://www.linkedin.com/in/dboyceman>
List-Unsubscribe: ***@***.***>
________________________________

**Comment by caver456** [comment]
Hi David, the drone techie (Ken) has indicated that he's in single-user
mode until his stuff gets farther along.  I did mention your interest
though, so when he emerges hopefully I can get you in touch.

I'd probably pass on that RFP, but thanks for thinking of the project.
Sounds more like a voice-to-text thing, and even after the text is
generated, straight-up transcription doesn't sound like it would be that
closely related to radiolog.
